### DISCUSSION TITLE
Which connection is correct to use load balancing?

### DISCUSSION THREAD

**Discussion by swocon** [SAP/node-rfc#278] [Q&A] [answered]
Hi all,
we want to use the library for an small external application.
To connect to SAP ERP we have to use load balancing.

So far we tried different configurations for the connection but nothing is working well. The error is all time "RFC_INVALID_PARAMETER" >> Parameter ASHOST, GWHOST, MSHOST or PORT is missing.
But to be honest I don't know what should be wrong...
The values of each parameter are correct. We using these for other libraries as well.

**FIRST CONNECTION**
sysnr
client
mshost
saprouter
user
passwd
lang
group

**SECOND CONNECTION**
as first connection + MSSERV

I have checked the sapnwrfc.ini of the NWRFC-SDK. Ther is no parameter PORT defined...

Can somebody help me to get the right list of parameters for connection with message server (load balancing) with this library?

Regards
Christian

**Comment by bsrdjan** [comment, marked as answer]
Please check https://github.com/SAP/node-rfc/issues/240#issuecomment-1030830175
